### DISCUSSION TITLE
i can't run this application

### DISCUSSION THREAD

**Discussion by yosoyebj** [alejandroautalan/pygubu-designer#66] [General]
hi. i installed pygubu and pygubu designer exactly like what you guys explained in the description. but when I came to run it, the linux terminal shows as command not found. what's the reason for it?

**Comment by alejandroautalan** [comment]
Hello @yosoyebj, thanks for trying pygubu.

Installing pygubu-designer with pip as a **normal user**, the pygubu-designer script is located at

    $HOME/.local/bin

So, you can add that path to your environment variables, or run pygubu-designer by typing this command in the console:

    ./.local/bin/pygubu-designer

Another alternative is to run pygubu-designer as a module with this command:

    python3 -m pygubudesigner

Let me know if this helps.
Regards.

**Comment by yosoyebj** [comment]
Yes.. it helped
